binder/environment.yml

```diff
@@ -1,7 +1,6 @@
 name: napari
 channels:
   - conda-forge
-  - ome
 dependencies:
   - zeroc-ice36-python
```

Comment:
```suggestion
```
The conda-forge name is `zeroc-ice`, though you can omit it since `omero-py` [handles the dependency including version constraint](https://github.com/conda-forge/omero-py-feedstock/blob/cbddd062c7df01250f3865a6ab69e0120ae6430e/recipe/meta.yaml#L32).

Reply:
thanks. This is older stuff.